runelite-client/src/test/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPluginTest.java

```diff
@@ -427,4 +428,29 @@ public void testZanarisFairyRing()
 			menu("Last-destination (AIQ)", "Fairy ring", MenuAction.GAME_OBJECT_SECOND_OPTION),
 		}, argumentCaptor.getValue());
 	}
+
+	@Test
+	public void testFriendsListDeleteEnabled()
+	{
+		when(config.swapFriendsListDelete()).thenReturn(true);
+
+		entries = new MenuEntry[] {
+			menu("Cancel", "", MenuAction.CANCEL),
+			menu("Lookup", "player", MenuAction.WIDGET_THIRD_OPTION),
+			menu("Delete", "player", MenuAction.WIDGET_SECOND_OPTION),
+			menu("Message", "player", MenuAction.WIDGET_FIRST_OPTION)
+		};
+
+		menuEntrySwapperPlugin.onMenuOpened(new MenuOpened());
```

Comment:
What is this event for?

Reply:
indeed, not necessary. I added it because my test initially kept failing, but that was because the menu target argument was wrong, I forgot to remove this line.